data/coalition/heliarch intro.txt

```diff
@@ -0,0 +1,1843 @@
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+
+mission "Heliarch Investigation 1"
+	name "Transport Heliarch Agents"
+	description "Bring <bunks> Heliarch agents to <destination>."
+	minor
+	passengers 3
+	to offer
+		random < 65
+		"coalition jobs" >= 70
+		has "Coalition: Contributor: done"
+		not "Resistance: Questions: done"
+	source
+		near "Quaru" 4 10
+	destination "Ring of Friendship"
+	on offer
+		conversation
+			`A group of Heliarch agents rapidly approach you as you step onto the spaceport, looking rather agitated as they stand in front of you.`
+			`	"Looking for work, the human visitor is?" the Saryd asks.`
+			`	"To the <destination>, a means of transport we need," the Kimek follows.`
+			`	"<payment>, we can offer," the Arach finishes.`
+			choice
+				`	"Of course, I'll show you to my ship."`
+				`	"Sorry, I'm not looking for any work currently."`
+					decline
+			`	They thank you for agreeing to deliver them to the <planet>, and introduce themselves; the Saryd is called Iplora, the Kimek is Kirrie, and the Arach is known as Tugroob. You help them settle in their bunks, wondering why they seemed so rushed.`
+				accept
+	on complete
+		"coalition jobs" ++
+		payment 90000
+		conversation
+			`The three Heliarchs disembark, heading immediately across the spaceport, but not before Iplora hands you <payment>. "Speak with our superior, we must. Need your services again, we likely will, so if interested in further work, you are, look for us in the spaceport, you should."`
+
+mission "Heliarch Investigation 2"
+	name "Heliarch Investigation"
+	description "Take the Heliarch agents to the marked planets so that they can investigate signs of criminal activity."
+	passengers 3
+	to offer
+		has "Heliarch Investigation 1: done"
+		not "Resistance: Questions: done"
+	source "Ring of Friendship"
+	destination "Ring of Friendship"
+	to complete
+		has "HI2-MP: done"
+		has "HI2-SF: done"
+		has "HI2-SS: done"
+		has "HI2-FS: done"
+		has "HI2-IW: done"
+		has "HI2-RB: done"
+	on offer
+		conversation
+			`The Heliarch trio you brought here approaches you when you enter the spaceport, Kirrie being the first to greet you, "Captain <last>! Glad I am, that still around, you are."`
+			`	"Accompany us, would you? A job in mind for you, our superior has." Iplora says.`
+			choice
+				`	"I don't see why not, lead the way."`
+				`	"Not this time, I'm really busy now."`
+					decline
+			`	The trio escorts you through a number of restricted passageways, eventually leading you to a meeting room of sorts with about a dozen other Heliarchs, one of whom motions for you to sit with them.`
+			`	You take a seat before one of the Kimek there speaks. "The honorable Captain, we salute. Make you feel welcome, I, Consul Iekie of the Heliarch council, hope we will."`
```

Comment:
They could be keeping an eye on you specifically because the the player is the first human visitor in a while and shows some interest in helping the coalition. So the consuls could see great use in them.

Reply:
True, but if there are any intermediate ranks it might be sensible for the Consul to have a "right-hand man" so to speak who is one rank down, who might handle any of the busy work associated with tracking and communicating with you.

Reply:
Consul is the highest rank.
This Intro introduces you to 3 of them, one in each ring, and each being representative of a different aspect of the Heliarch government.
Out of the 3, Iekie is the most involved with standard government business for the Heliarchy, but also is one of those in charge of investigating people and places that they believe might be involved with the Lunarium.

Considering that in one of the other chains where you meet a consul (Plortub on Ring of Power) you only ever meet at about halfway through the chain, and the last consul (Aulori on Ring of Wisdom) only ever shows up on another chain, *after* you've already done a lot of work for him via assisting his teams, I don't think it's an issue that Iekie, the more government-focused of the 3 consuls, is met early on on her chain.
You meet Iekie right at the start, Plortub at about the middle, and Aulori way later, on their respective chains.

I also don't want to go on creating stepping stone characters before the player can interact with the consuls proper, since it's mainly the consuls that the player will interact with during the campaign anyway.
The player is an exceptional figure in the Coalition, being an outsider, and having a JD of their own, so even if for normal aspiring Heliarchs meeting a consuls might be ways off, I don't think that should also apply to the player in these missions.